src/App.tsx

```diff
@@ -1,14 +1,51 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { Todo } from './types/Todo';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [filterStatus, setFilterStatus] = useState('all');
+  const [selectedTodoId, setSelectedTodoId] = useState(0);
+  const [todos, setTodos] = useState<Todo[]>([]);
+
+  useEffect(() => {
+    getTodos().then(todosFromServ => setTodos(todosFromServ));
```

Comment:
```suggestion
    getTodos().then(setTodos);
```

shorten it and add error handling